pkg/interceptors/github/github.go

```diff
@@ -59,6 +59,10 @@ func (w *Interceptor) Process(ctx context.Context, r *triggersv1.InterceptorRequ
 	}
 	// Validate secrets first before anything else, if set
 	if p.SecretRef != nil {
+		// Check the secret to see if it is empty
+		if p.SecretRef.SecretKey == "" {
+			return interceptors.Fail(codes.FailedPrecondition, "interceptor secretRef.secretKey is empty")
```

Comment:
How about adding interceptor name also in the error message it will be very easy for debugging 

ex: `github interceptor secretRef.secretKey is empty`

same for bitbucket and gitlab

Reply:
Great idea. I'll go ahead and make this update today.